packages/store/addon/-private/system/references/reference.ts

```diff
@@ -31,6 +34,10 @@ function isResourceIdentiferWithRelatedLinks(
 
 export const INTERNAL_MODELS = new WeakMap<Reference, InternalModel>();
```

Comment:
Sort of along the same lines as the other question - The API changed for creating References from InternalModel to ids.  However, do we want to also change our caching layer as well with this refactor?  Or does this seem right to keep as is (various places in belongs-to/has-many reference IM specific properties)?

Reply:
Replacing the caching layer is 👍